Formula/readtools.rb

```diff
@@ -1,8 +1,8 @@
 class Readtools < Formula
-  desc "Standardizing sources for sequencing data"
+  desc "A Universal Toolkit for Handling Sequence Data from Different Sequencing Platforms"
```

Comment:
Note that I changed the description for the new repository description (matching the manuscript).

Reply:
It is too long, though, with its 83 characters. brew audit also does not like beginning with indefinite articles and discourages filler terms like "framework", "program", and, I guess, also "toolkit", with little meaning beyond what is already clear from the context. I think "Handling Sequence Data from Different Sequencing Platforms" might be a description in the homebrew spirit, but is a bit boring. Maybe "Universal Toolkit for Handling Sequence Data from Different Platforms" is close enough?

Reply:
I will go for the hombrew spirit! Nothing better than following conventions for a program that it is supposed to be for "standardizing" formats...hahaha. Next commit have `Handling Sequence Data from Different Sequencing Platforms` instead.

Thanks for the feedback!